README.md

```diff
@@ -14,20 +14,22 @@
 [![ktlint](https://img.shields.io/badge/code%20style-%E2%9D%A4-FF4081.svg)](https://ktlint.github.io/)
 [![Chat on Telegram](https://img.shields.io/badge/Chat%20on-Telegram-brightgreen.svg)](https://t.me/joinchat/AAAAAFDg-ipuZFGyBGPPeg)
 
-## (!) See [diKTat codestyle](info/diktat-kotlin-coding-style-guide-en.md) first.
+## (!) See [diKTat codestyle](info/diktat-kotlin-coding-style-guide-en.md) first. This is a formal code style used by this linter.
 ## (!) Also see [the list of all rules supported by diKTat](info/available-rules.md).
-## (!) Have a look at [maven and gradle examples](https://github.com/akuleshov7/diktat-examples) of usage diKTat with plugins
+## (!) Have a look at [maven and gradle examples](https://github.com/akuleshov7/diktat-examples) of usage diKTat with plugins.
 
 DiKTat is a collection of [Kotlin](https://kotlinlang.org/) code style rules implemented
 as AST visitors on top of [KTlint](https://ktlint.github.io/).
 The full list of available supported rules and inspections is [here](info/available-rules.md).
 
 ## Run as CLI-application
-1. Install KTlint (until this [PR](https://github.com/pinterest/ktlint/pull/806) is merged you will need to use
- [KTlint fork](https://central.artipie.com/akuleshov7/files/ktlint)):
-   ```bash
-   $ curl -sSLO https://central.artipie.com/akuleshov7/files/ktlint && chmod a+x ktlint
-   ```
+1. Install KTlint manually: [here](https://github.com/pinterest/ktlint/releases)
+
+   **OR** use curl:
+    ```bash
+    curl -sSLO https://github.com/pinterest/ktlint/releases/download/0.37.1/ktlint && chmod a+x ktlint
```

Comment:
KtLint 0.38+ uses kotlin 1.4 and is not yet compatible with diktat

Reply:
Yes, I know, Amended yesterday before your comment :)